filesystem/android_super.ksy

```diff
@@ -0,0 +1,174 @@
+meta:
+  id: android_super
+  title: Android Dynamic Partitions metadata
+  application: Android
+  file-extension: img
+  license: CC0-1.0
+  ks-version: 0.8
+  endian: le
+
+doc: |
+  The metadata stored by Android at the beginning of a "super" partition, which
+  is what it calls a disk partition that holds one or more Dynamic Partitions.
+  Dynamic Partitions do more or less the same thing that LVM does on Linux,
+  allowing Android to map ranges of non-contiguous extents to a single logical
+  device. This metadata holds that mapping.
+
+doc-ref: https://source.android.com/devices/tech/ota/dynamic_partitions/implement
+doc-ref: https://android.googlesource.com/platform/system/core/+/master/fs_mgr/liblp/include/liblp/metadata_format.h
```

Comment:
This is actually invalid YAML, duplicate keys aren't allowed. Some parsers allow them, but the latter `doc-ref` will silently overwrite the first one in that case, it won't become an array.

The proper solution is to put these refs to a sequence, which is allowed since 0.9 (see https://github.com/kaitai-io/kaitai_struct/issues/269): 

```suggestion
doc-ref
 - https://source.android.com/devices/tech/ota/dynamic_partitions/implement
 - https://android.googlesource.com/platform/system/core/+/master/fs_mgr/liblp/include/liblp/metadata_format.h
```

And please also increment the top-level `ks-version` to 0.9.